### PR TITLE
Add backing-store cypress test to odf-console

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Signed-off-by: Timothy Asir Jeyasingh <tjeyasin@redhat.com>

### CODE REVIEW

File: cypress/tests/create-backing-store.spec.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  afterEach(() => {
+    cy.byLegacyTestID('actions-menu-button').click();
+    cy.byTestActionID('Delete Backing Store').click();
+    cy.byTestID('confirm-action').click();
+  });
+
+  beforeEach(() => {
+    cy.visit('/');
+    commonFlows.navigateToODF();
```
Comment (TimothyAsirJeyasing) [inline cypress/tests/create-backing-store.spec.ts:23]: Need to update in the commonFlows

https://github.com/red-hat-storage/odf-console/pull/87

**Review by TimothyAsirJeyasing — COMMENTED** [review]
Need to update the function details in cypress/views/common.ts

File: cypress/tests/create-backing-store.spec.ts (hunk 1 above)
Comment (vbnrh) [reply cypress/tests/create-backing-store.spec.ts:23]: Use `ODFCommon.visitStorageDashboard();`

**Pull request merged**